tools/reviewer/web/src/app/dashboard/reviews-panel/review/review-titlebar/reply-popup/reply-popup.component.ts

```diff
@@ -71,6 +72,20 @@ export class ReplyPopupComponent {
       reviewer.setNeedsAttention(false);
     }
 
+    // Add the message as a DiffThread
+    this.message = this.message.trim();
+    if (this.message) {
+      const diffThread: Thread = new Thread();
+      const comment: Comment = new Comment();
+      comment.setContent(this.message);
+      comment.setCreatedBy(this.authService.userEmail);
+      comment.setTimestamp(Date.now());
+      // Set isDone of Thread based on Action Required Checkbox
+      diffThread.setIsDone(!this.actionRequired);
+      diffThread.addComment(comment);
```

Comment:
I can see diff threads without any message:
https://i.imgur.com/fgAm1q2.jpg

How did you manage to add them?
Also, isn't it nonsense? They don't contain any useful information.

Reply:
they can be used to get the attention of other side or even approve the Diff. I just confirmed with Ofer.